api/v3/Award.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * @file
+ * Award Detail API file.
+ */
+
+/**
+ * Award.get API specification.
+ *
+ * @param array $spec
+ *   Description of fields supported by this API call.
+ */
+function _civicrm_api3_award_get_spec(array &$spec) {
+  $spec['managed_by'] = [
+    'title' => 'Award Manager',
+    'description' => 'Award Manager Contact ID',
+    'type' => CRM_Utils_Type::T_STRING,
+  ];
+
+  $spec['award_detail_params'] = [
+    'title' => 'Params for AwardDetail.Get API',
+    'description' => 'Array of parameters for AwardDetail.Get API',
+    'type' => CRM_Utils_Type::T_STRING,
+  ];
+
+  $spec['case_type_params'] = [
+    'title' => 'Params for CaseType.Get API',
+    'description' => 'Array of parameters for CaseType.Get API',
+    'type' => CRM_Utils_Type::T_STRING,
+  ];
+}
+
+/**
+ * Award.get API.
+ *
+ * @param array $params
+ *   API parameters.
+ *
+ * @return array
+ *   API result descriptor
+ */
+function civicrm_api3_award_get(array $params) {
```

Comment:
It will take me more time to understand and write tests, and I dont want to spend more time in this ticket. But definitely, I will start adding BE tests in the future.

Will add a ticket though 👍